### ISSUE TITLE
Master key to disable all sorts of annoyance due to hints/autocomplete

### ISSUE THREAD

**Issue by SandboChang** [spyder-ide/spyder#13837]
I like Spyder a lot with it clean interface, but lately with the new version it is loaded with numerous annoyance that pops up here and there.

I tried to disable them one by one in the completion tab, but it is just hard to know the option location for every single of them.
Would it be possible to just provide a master switch to turn them all off?

**Comment by steff456** [comment]
Hi @SandboChang,

We can discuss this with the team for including this for Spyder 5, can you please give me a list of the pop ups that you want to remove?

**Comment by SandboChang** [comment]
Thanks for getting back to me, and sorry for the late reply.

I tried to look up the settings, I mostly want to disable those which shows up as I type. Those which show up when I hover the mouse over are less obstructive. 

I tried to change many settings, now I cannot tell exactly which was ON/OFF by default, but here are probably some:
Preference-->Completion and linting
-->Completion
- Enable code completion
- Enable code snippets
- Enable Kite
- Enable fallback completions
